changelog.txt

```diff
@@ -3,3 +3,4 @@
 * Fixes bug in Firestore emulator where property paths with special characters would cause errors due to ClassNotFound exceptions.
 * Fixes bug in Firestore emulator where auto-id allocation only worked once per collection.
 * Adds REST API to Firestore emulator for setting security rules.
+* Made all body-parser configs use 10mb request limits
```

Comment:
Can this be changed to: "Functions emulator now limits request sizes to 10mb."? Without the context of what feature is changing, this stand-alone sentence is hard to understand.

@samtstern if you have any better wording, feel free to share.

Be sure to finish the sentence with a period (`.`), please.